salt/output/highstate.py

```diff
@@ -141,6 +137,12 @@ def _format_host(host, data):
                ret['result'] and not schanged:
                 continue
 
+            # Skip this state if state_verbose is False, the result is True and
+            # there were no changes made
+            if not __opts__.get('state_verbose', False) and \
+               ret['result'] and not schanged:
+                continue
```

Comment:
how did the linter miss this one?!?
Can you please fix the spacing here @anlutro ?